snapcraft/application.py

```diff
@@ -62,6 +62,25 @@
 }
 
 
+def _get_esm_error_for_base(base: str) -> None:
```

Comment:
Could you do a quick pair of unit tests for this?

Reply:
Added